### DISCUSSION TITLE
Newbie struggling to add a new page and navigate to it

### DISCUSSION THREAD

**Discussion by Zuuai** [vercel/community#5001] [Help]
### Summary

I jumped in at the deep and. I have managed to get the subscription app to work with Stripe. I would now like to give access to the app that my subscribers have paid for. This will consist of some Javascript files that like to a Voiceflow app.

Please I really need hep to create a new page where I can embed the VF app and give the subscribers a means to navigate to it.

` <Card
          title="Your Plan"
          description={
            subscription
              ? `You are currently subscribed to ${subscription?.prices?.products?.name}.`
              : 'You are not currently subscribed to any plan.'
          }
         footer={<ManageSubscriptionButton session={session} />}
        >
          <div className="mt-8 mb-4 text-xl font-semibold">
            {subscription ? (
              `${subscriptionPrice}/${subscription?.prices?.interval}`
            ) : (
              <Link href="/">View Plans</Link>
            )}
          </div>
            <div className="mt-8 mb-4 text-xl font-semibold">
            {subscription ? (
              <Link href="/account/dsa">Access Seller Assistant</Link>
            ) : (
              <Link href=""></Link>
            )}
          </div>
        </Card>`

The Access Seller Assistant link does not work. I have created a dsa.js file as follows:

`import Link from 'next/link';

export default function dsa() {
  return (
    <>
      <h1>Seller Assistant</h1>
      <h2>
        <Link href="/account/dsa">Access Seller Assistant</Link>
      </h2>
    </>
  );
}`

Ii have followed the example on NextJS docs but it refers to the Pages directory which does not exist in this project.

### Example

https://loadscience.app/account

### Steps to Reproduce

No steps to produce

**Comment by amyegan** [comment]
Hey @Zuuai! It looks like you're using the App Router rather than the Pages Router. I believe this is the page you need: https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts

You can switch the Next.js docs to show App Router or Page Router info by changing which one is selected at the top of the documentation menu.

<img width="975" alt="router selector in documentation menu with app router selected" src="https://github.com/vercel/community/assets/8247105/93dc14cd-1c66-4e19-b573-9b8f9d07f2fe">



There is also a learning course available to help you get started with the App Router and other Next.js 14 features: https://nextjs.org/learn
